frontend/src/Component/JobOfferCard.tsx

```diff
@@ -1,17 +1,21 @@
 import {useNavigate} from "react-router-dom";
 import {JobOffer} from "../types/JobOffer.ts";
-type jobOfferCardProps={
-    jobOffer:JobOffer
+
+type jobOfferCardProps = {
+    jobOffer: JobOffer
 }
-export default function JobOfferCard({jobOffer}:Readonly<jobOfferCardProps>){
-    const navigate=useNavigate();
-    function navigateToJobOfferDetails(){
-        navigate("/joboffer/"+jobOffer.id)
+export default function JobOfferCard({jobOffer}: Readonly<jobOfferCardProps>) {
+    const navigate = useNavigate();
+
+    function navigateToJobOfferDetails() {
+        navigate("/joboffer/" + jobOffer.id)
     }
-    function navigateToNewApplicationPage(){
-        navigate("/newApplication/"+jobOffer.id)
+
+    function navigateToNewApplicationPage() {
+        navigate("/newApplication/" + jobOffer.id)
     }
```

Comment:
This could be part of a refactor. Since the functions are doing the same, just the first path of the resource is changing, you could simplify it with just one function and passing the path as variable

```javascript
function navigateTo(path: string): void {
        navigate(`/${path}/${jobOffer.id}`)
}
```